src/app/History.ts

```diff
@@ -7,6 +7,7 @@ const history = createBrowserHistory({ basename: baseName });
 export default history;
 
 export enum URLParams {
+  AGG_OPERATOR = 'aggOperator',
```

Comment:
Perhaps avoid another abbreviation by using 'aggregator'?  And aggregator is basically an aggregation operator, it's something that aggregates, creating an aggregation :)  if you like the word I'd suggest carrying it through the code.

Reply:
yes, I didn't think about that one but it makes sense.